server/server/src/static_files.rs

```diff
@@ -59,8 +59,8 @@ async fn plugins(req: HttpRequest, settings: Data<Settings>) -> Result<HttpRespo
     let filename = path.get("filename");
 
     let file = PluginFileService::find_file(&settings.server.base_dir, plugin, filename)
-        .map_err(|err| InternalError::new(err, StatusCode::INTERNAL_SERVER_ERROR))?
-        .ok_or(std::io::Error::new(ErrorKind::NotFound, "Plugin not found"))?;
+        .unwrap()
+        .unwrap();
```

Comment:
> - add an unwrap (as per [this comment](https://github.com/openmsupply/open-msupply/pull/2146#discussion_r1315305629))

With this comment I was referring to above lines:
![Screenshot 2023-10-05 at 12 10 05 AM](https://github.com/openmsupply/open-msupply/assets/26194949/816be447-3a0b-4c3d-9c7d-3bc3dae9efb8)

Also, from the same comment, the suggestion "#[get(r#"/{filename:.*\..+$}"#)]" and https://actix.rs/docs/extractors/#path, was for something like this:

https://github.com/openmsupply/open-msupply/compare/2191-api-feedback...2191-api-feedback(Type-safe-path-extractions)

Removed filename optional as we don't seem to use it ? 
https://github.com/openmsupply/open-msupply/blob/7ea8535f6a69028e3852b06590b2d00c5b52fa9a/client/packages/host/src/plugins/useInitPlugins.ts#L127

Btw I didn't test it, a couldn't build plugins in the origin/feature/front-end-plugins-example, yarn build-plugins from root get this:
![Screenshot 2023-10-05 at 12 41 18 AM](https://github.com/openmsupply/open-msupply/assets/26194949/4e93931b-3666-43fd-8c72-d6dd5837705a)

When trying to build individual get this:
![Screenshot 2023-10-05 at 12 42 34 AM](https://github.com/openmsupply/open-msupply/assets/26194949/d4fc92c3-51f3-4acc-873b-28a8297eff99)

Btw the server plugins directory doesn't exist, need to be manually created (fyi, that's another errorI think it's good to create it with scripts, especially for newcomers to plugins)

Reply:
Thanks @andreievg have merged your suggestion, reverted the unwraps and fixed the example branch again. Too many active branches 🙄